catching-non-exception:27:7::"Catching an exception which doesn't inherit from Exception: MyException"
catching-non-exception:33:7::"Catching an exception which doesn't inherit from Exception: MyException"
catching-non-exception:33:7::"Catching an exception which doesn't inherit from Exception: MySecondException"
catching-non-exception:54:7::"Catching an exception which doesn't inherit from Exception: None"
catching-non-exception:54:7::"Catching an exception which doesn't inherit from Exception: list()"
catching-non-exception:59:7::"Catching an exception which doesn't inherit from Exception: None"
catching-non-exception:72:7::"Catching an exception which doesn't inherit from Exception: 4"
catching-non-exception:72:7::"Catching an exception which doesn't inherit from Exception: None"
catching-non-exception:72:7::"Catching an exception which doesn't inherit from Exception: list([4, 5, 6])"
catching-non-exception:85:7::"Catching an exception which doesn't inherit from Exception: NON_EXCEPTION_TUPLE"
catching-non-exception:103:7::"Catching an exception which doesn't inherit from Exception: object"
catching-non-exception:108:7::"Catching an exception which doesn't inherit from Exception: range"
